samples/snippets/encrypt_symmetric.py

```diff
@@ -45,7 +49,30 @@ def encrypt_symmetric(project_id, location_id, key_ring_id, key_id, plaintext):
     key_name = client.crypto_key_path(project_id, location_id, key_ring_id, key_id)
 
     # Call the API.
-    encrypt_response = client.encrypt(request={'name': key_name, 'plaintext': plaintext_bytes})
+    encrypt_response = client.encrypt(
+      request={'name': key_name, 'plaintext': plaintext_bytes, 'plaintext_crc32c': plaintext_crc32c})
+    
+    # Optional, but recommended: perform integrity verification on encrypt_response:
+    if not encrypt_response.verified_plaintext_crc32c:
+        raise Exception('The request sent to the server was corrupted in-transit.')
+    if not encrypt_response.ciphertext_crc32c == crc32c(encrypt_response.ciphertext):
+        raise Exception('The response received from the server was corrupted in-transit.')
+    # End integrity verification
+    
     print('Ciphertext: {}'.format(base64.b64encode(encrypt_response.ciphertext)))
     return encrypt_response
+  
+def crc32c(data):
```

Comment:
Not sure if https://pypi.org/project/google-crc32c/ is of interest here

Reply:
Thanks for pointing out! This requires Python >=3.5, while the open source crcmod supports Python2 and 3. 
I'll add a comment to link to this library.

Reply:
This library is also 3.6+ now (see [setup.py](https://github.com/googleapis/python-kms/blob/09c5fed9629cda301614938477f8bced2abaf34c/setup.py#L80)), so it may be alright to take the dependency.

Reply:
Looking at this closer: Using crcmod modules is the current recommendation by Google Cloud GCS (https://cloud.google.com/storage/docs/hashes-etags#_CRC32C) and also aligns with the internal implemntation in Cloud KMS's server and clients. 

So I prefer to leave the crcmod suggestion in place.

Reply:
@crwilcox Is there any guidance on cloudsite on when/how to use `google-crc32c`?